python/cuml/cluster/hdbscan/prediction.pyx

```diff
@@ -129,9 +142,12 @@ def all_points_membership_vectors(clusterer):
     Parameters
     ----------
     clusterer : HDBSCAN
-         A clustering object that has been fit to the data and
+        A clustering object that has been fit to the data and
         had ``prediction_data=True`` set.
 
+    batch_size : int, optional, default=0
+        Distance based membership is computed in batches to fit on the device. If not specified, or set to 0, distance based membership is computed at once for all points in the training data.
```

Comment:
I think this could be worded a little bit better to highlight the purpose of this setting, maybe mention "points in the training data" closer to the beginning?  Something like "Lowers memory requirement by computing distance-based membership in smaller batches of points in the training data. Batch size of 0 uses all of the training points, batch size of 1000 computes distances for 1000 points at a time."